### PR TITLE
Bump sdk version to version 1.11.0

### PR DESCRIPTION
**Pull request by v-momoin**
Appcenter Service delivery Team

Things to consider before you submit the PR:

* [x] Has `CHANGELOG.md` been updated?
* [x] Are tests passing locally?
* [x] Are the files formatted correctly?
* [ ] Did you add unit tests?
* [ ] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

Working on Apple SDK release feature for the month of November. https://msmobilecenter.visualstudio.com/Mobile-Center/_workitems/edit/52662

## Related PRs or issues

List related PRs and other issues.

## Misc

Add what's missing, notes on what you tested, additional thoughts or questions.

### CODE REVIEW

**Comment by v-momoin** [comment]
Executed the below steps from the Apple SDK doc.

Verify markdown documents and update versions

1.CHANGELOG.md , add any public SDK changes (behavior/code) introduced in this version. (Note: future releases will come with an updated changelog file already as they will be required for each PR). Remove any marks like (Under active development) for current version.
2.Check version in all .jazzy.yaml files from the Documentation  folder.
3.Check version in the Config/Version.xcconfig  file.
4.Update the AppCenter.podspec  file.
5.Submit a PR against the develop branch for above changes. This should be the last PR/commits into the develop branch for the version.

**Comment by ElektrojungeMS** [comment]
Please add the following to `Changelog.md` as part of this PR under the AppCenter Analytics section:

```
[Feature] Add support for typed properties. Note that these APIs still convert properties back to strings on the App Center backend. More work is needed to store and display typed properties in the App Center portal. Using the new APIs now will enable future scenarios, but for now the behavior will be the same as it is for current event properties.
[Feature] Preparation work for a future change in transmission protocol and endpoint for Analytics data. There is no impact on your current workflow when using App Center.
```

**Comment by v-momoin** [comment]
Updated the Changelog.md as per the comments form Benjamin.

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,8 @@
 ### AppCenterAnalytics
 
 * **[Feature]** Add new trackEvent APIs that take priority (normal or critical) of event logs. Events tracked with critical flag will take precedence over all other logs except crash logs (when AppCenterCrashes is enabled), and only be dropped if storage is full and must make room for newer critical events or crashes logs.
+* **[Feature]** Add support for typed properties. Note that these APIs still convert properties back to strings on the App Center backend. More work is needed to store and display typed properties in the App Center portal. Using the new APIs now will enable future scenarios, but for now the behavior will be the same as it is for current event properties.
+* **[Feature]** Preparation work for a future change in transmission protocol and endpoint for Analytics data. There is no impact on your current workflow when using App Center.
```
[the remark this answers is not in the document]
Comment (guperrot) [reply CHANGELOG.md]: Discussed offlines, the 2 added points can be removed.

[the remark this answers is not in the document]
Comment (v-momoin) [reply CHANGELOG.md]: Updated as discussed.

**Pull request merged**